rust-lang/this-week-in-rust
# Dark theme accessibility

## Why
For #4031
In dark mode, `<code>` elements are red-on-black which is difficult to read

## What Changed
- Added a `--code-color` variable with existing color
- Copied color from newsletter dark theme

Previous contrast ratio: [2.59:1](https://webaim.org/resources/contrastchecker/?fcolor=2A2A2A&bcolor=C7254E)
New contrast ratio: [5.48:1](https://webaim.org/resources/contrastchecker/?fcolor=2A2A2A&bcolor=EB7F98)

## Testing
- Build the site with the rusted theme: `pelican --delete-output-directory content`
- Check `<code>` elements in one of the posts.  The 'Updates from the Rust Project' section has several
